diagnostic.md

```diff
@@ -5,74 +5,82 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```md
-// your response here
+The purpose of the backend is to manage and persist data to the front end client.
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```md
-// your response here
+Model
 ```
 
 Which layer in the MVC pattern communicates with the model?
 
 ```md
-// your response here
+Controller
 ```
 
 Why don't we use views in our interpretation of the MVC pattern?
 
 ```md
-// your response here
+???
```

Comment:
We don't use views on our back-end because we manage all that on the front-end! This is how SPAs function, to lessen the amount of requests made to the back-end.